jax/experimental/export/_export.py

```diff
@@ -106,11 +103,15 @@ def custom_call(cls, target_name: str) -> DisabledSafetyCheck:
 
   @classmethod
   def shape_assertions(cls) -> DisabledSafetyCheck:
-    """Allows invocations with shapes that do not meet the constraints.
+    """Allowed invocations with shapes that do not meet the constraints.
```

Comment:
Should we say this is now a noop in the docstring as well?

Reply:
Done